src/qml/BadLayerItem.qml

```diff
@@ -101,6 +102,8 @@ Page {
       Layout.minimumHeight: contentHeight
       Layout.maximumHeight: contentHeight
       Layout.topMargin: 5
+
+      onLinkActivated: Qt.openUrlExternally(link)
```

Comment:
Oh and we need to add the parameter here, Qt plans to eventually turn of parameter injection altogether. It should be

(link) => { Qt.openUrlExternally(link); }